src/tools/genpolicy/rules.rego

```diff
@@ -1214,6 +1235,38 @@ allow_storage(p_storages, i_storage, bundle_id, sandbox_id) if {
     # TODO: Check Mount Point, Source, Driver Options, etc.
     print("allow_storage with image_guest_pull: true")
 }
+allow_storage(p_storages, i_storage, bundle_id, sandbox_id) if {
+    print("allow_storage emptyDir_encrypted blk: start")
+
+    some p_storage in p_storages
+    p_storage.driver == ""
+
+    "blk" == i_storage.driver
```

Comment:
feel free to close: Do we not usually place the i_ to the left in string comparisons? Example `i_storage.fstype == "overlay"` and others. If you intend to adjust, there's a few more occurrences below

Reply:
This is intended to match the first `allow_storage()` rule but open to other opinions.